src/plugins/intel_gna/frontend/quantized_layer_params.hpp

```diff
@@ -85,7 +85,8 @@ struct QuantizedLayerParams {
     Quantization _weights_quant;
     Quantization _bias_quant;
 
-    bool lowPrecision = false;
+    bool _weights_low_precision = false;
+    bool _inputs_low_precision = false;
```

Comment:
I leading "_" convention used anywhere in the OV core?
if not I would not use it anymore.